styles/widgets/material/scheduler.material.less

```diff
@@ -162,13 +187,17 @@
     .dx-scheduler-vertical-group-table-mixin(1px);
 }
 
+.dx-scheduler-appointment-content {
```

Comment:
🤔

Reply:
We need to rewrite the 'common' styles. All styles are not rewritten by 194-202 lines.